metadata.rb

```diff
@@ -24,7 +24,7 @@
 chef_version     '>= 12' if respond_to?(:chef_version)
 depends          'compat_resource'
 
-gem              'oneview-sdk' if respond_to?(:gem)
+gem              'oneview-sdk', '=1.0.0' if respond_to?(:gem)
```

Comment:
@jsmartt This is here because of the backward compatibility issues. The version is being fixed there so the cookbook won't get any SDK version that is not compatible with the current implementation (Notice the current cookbook implementation will not work with SDK version 2.0.0).